### PR TITLE
fixed clearing categories in market creation

### LINKED ISSUE #3337: Filters not clearing out after discarding draft in market creation
When creating a custom market, filling in the drop downs of Market Categories and then discarding it.  When user goes to create a custom market the next time, those Market Category filters are still selected

What I think I saw was go through Yes No market flow to second form then nav to market list. 
then back to market creation categories didn't get cleared out.

### LINKED ISSUE #4022: Outcomes dont disappear in market creation from previous created market
I created a custom market.  Went back on to create another custom market and the outcomes from market1 where still in the new market creation

### PR DESCRIPTION
**Pull request by bthaile**
issue where default_state was getting updated so when new-market reducer was cleared the categories were left. 

closes https://github.com/AugurProject/augur/issues/3337
fixes https://github.com/augurproject/augur/issues/4022

### CODE REVIEW

**Review by phoebemirman — CHANGES_REQUESTED** [review]
Screen Shot 2019-10-08 at 1 28 16 PM
getting this crash

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🔮

 This link will always point to this PR's latest changes: https://peek.run/7g6hvg9

File: packages/augur-ui/src/modules/markets/reducers/new-market.ts [hunk 1]
```diff
@@ -7,73 +7,17 @@ import {
...
-  orderBookSorted: {}, // for order book table
-  minPrice: '0',
-  maxPrice: '1',
-  minPriceBigNumber: ZERO,
-  maxPriceBigNumber: ONE,
-  initialLiquidityDai: ZERO,
-  initialLiquidityGas: ZERO,
-};
-
+const bob = JSON.parse(JSON.stringify(EMPTY_STATE));
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/markets/reducers/new-market.ts]: did you mean to keep this

Comment (bthaile) [reply packages/augur-ui/src/modules/markets/reducers/new-market.ts]: yes, only way I could make sure default state was actually default state.

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**